packages/client/src/components/Guidance/Guidance.tsx

```diff
@@ -11,14 +12,32 @@ export const Guidance: React.FC<{
 	isVisible: Maybe<boolean>
 	step?: Step
 }> = memo(function Instructions({ isVisible, step }) {
+	const [markdown, setMarkdown] = useState('')
+
+	useEffect(() => {
+		setMarkdown('')
```

Comment:
I would put this in an else clause if there is no markdownPath. If there is an error and concern about old state you can also put this in the catch.